includes/AMP/Meta_Sanitizer.php

```diff
@@ -0,0 +1,115 @@
+<?php
+/**
+ * Class Meta_Sanitizer.
+ *
+ * @package   Google\Web_Stories
+ * @copyright 2020 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://github.com/google/web-stories-wp
+ */
+
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Web_Stories\AMP;
+
+use AmpProject\Attribute;
+use AmpProject\Tag;
+use AMP_Meta_Sanitizer;
+
+/**
+ * Meta sanitizer.
+ *
+ * Sanitizes meta tags found in the header.
+ *
+ * This version avoids using amp_get_boilerplate_stylesheets().
+ *
+ * @see amp_get_boilerplate_stylesheets()
+ * @see AMP_Meta_Sanitizer
+ *
+ * @since 1.1.0
+ */
+class Meta_Sanitizer extends AMP_Meta_Sanitizer {
```

Comment:
We should have made `AMP_Meta_Sanitizer` a `final` 😬

Reply:
I can copy the whole class if that‘s more desirable?

Reply:
Nah, I think the ship has sailed on that already.